en/installation.rst

```diff
@@ -59,24 +57,32 @@ You should at least have got installed PHP 5.5.9 (CLI) or higher.
 Your webserver's PHP version must also be of 5.5.9 or higher, and should best be
 the same version your command line interface (CLI) PHP version is of.
 
+Installing Composer
+-------------------
+
 CakePHP uses `Composer <http://getcomposer.org>`_, a dependency management tool,
 as the officially supported method for installation.
 
-First, you'll need to download and install Composer if you haven't done so
-already. If you have cURL installed, it's as easy as running the following:
+Installing Composer on Linux and Mac OS X :
 
-.. code-block:: bash
+#. Run the installer script as described in the
+   `official Composer documentation <https://getcomposer.org/download/>`_ and
+   follow the instructions to install Composer.
+#. Execute the following command to move the composer.phar to a directory that
+   is in your path::
 
-    curl -s https://getcomposer.org/installer | php
+       mv composer.phar /usr/local/bin/composer
 
-Or, you can download ``composer.phar`` from the
-`Composer website <https://getcomposer.org/download/>`_.
+Installing Composer on Windows :
 
 For Windows systems, you can download Composer's Windows installer
```

Comment:
@antograssiot true, I'll add bullet in a future PR. The main composer page does not explain to move the bin in the path so I think this can help